backend/src/controllers/SessionController.ts

```diff
@@ -0,0 +1,52 @@
+import { SessionDataObject, SessionObject } from "../util/types/sessions";
+import jwt from "jsonwebtoken";
+import Duxcore from "../Duxcore";
+
+export class SessionController {
+	public id: string;
```

Comment:
formatting is still broken

Reply:
please elaborate

Reply:
you use the tab symbol instead of 2 spaces

Reply:
configure prettier js or eslint to format the code properly

Reply:
I'll be adding one once I'm done with what I am doing today and I am going to go through and set it up manually.